scripts/create-database.js

```diff
@@ -0,0 +1,48 @@
+const { Client } = require('pg');
+const path = require('path');
+
+const envName = process.argv.slice(2)[0];
+
+const loadEnv = envName => {
+  const { NODE_ENV } = process.env;
+  if (NODE_ENV != 'production') {
+    const envFile = envName === 'test' ? '../.env.test' : '../.env';
+
+    require('dotenv').config({
+      path: path.join(__dirname, envFile),
+    });
+
+    const databaseName = process.env.PGDATABASE;
+
+    delete process.env.PGDATABASE;
+
+    return databaseName;
```

Comment:
As I said on the last review, common functionality can be grouped together without the extra clear lines
```
    const databaseName = process.env.PGDATABASE;
    delete process.env.PGDATABASE;
    return databaseName;
```